graphql-test/src/main/java/org/jahia/test/graphql/GraphQLPublicationTest.java

```diff
@@ -183,11 +186,27 @@ private void testPublication(boolean publishSubNodes) throws RepositoryException
         JSONObject mutationResult = result.getJSONObject("data").getJSONObject("jcr").getJSONObject("mutateNode");
         Assert.assertTrue(mutationResult.getBoolean("publish"));
 
+        JSONObject expectedResult = executeQuery(""
+                + "query {"
+                + "  jcr(workspace: LIVE) {"
+                + "     nodeByPath(path: \"/sites/"+ siteName + "/testList\") {"
+                + "         property(name: \"j:published\") {"
+                + "            value"
+                + "         }"
+                +"      }"
+                +"   }"
+                +"}"
+        );
+        JSONObject queryResult = expectedResult.getJSONObject("data").getJSONObject("jcr").getJSONObject("nodeByPath").getJSONObject(
+                "property");
+        Assert.assertTrue("The node: '/sites/" + siteName + "/testList' should have been published", queryResult.getBoolean("value"));
```

Comment:
The assert will stop the execution if it fails and won't give more information (it won't go into debugSession). You can also maybe execute the same query in default (since it's quite clear the node does not exist in live when it fails)

Reply:
I see. I'll move it because I thought it'll continue with the test even if the assert at line 202 fails